### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 006080**
Nataliya Rodionova present

### CODE REVIEW

**Comment by 006080** [comment]
Here I added socials for our Footer

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ npm install
 ```
 
 ```bash
+
```
Comment (miklesch) [inline README.md:11]: can be deleted

File: src/components/FavoriteStay/FavoriteStay.jsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import React from "react";
+
+
```
Comment (miklesch) [inline src/components/FavoriteStay/FavoriteStay.jsx:2]: why deleted? correct empty space.

File: src/components/Footer/Footer.jsx
Comment (miklesch) [inline src/components/Footer/Footer.jsx:1]: The pullrequest is named Update README, but you add a feature to the footer component. I would recommend to change the name of the PR at the headline of your PR.

File: src/pages/ProductPage.jsx [hunk 3]
```diff
@@ -2,10 +2,10 @@ import ReservationCard from "../components/ReservationCard/ReservationCard";
 import FavoriteStay from "../components/FavoriteStay/FavoriteStay";
 import HostSummary from "../components/HostSummary/HostSummary";
 import hostImage from "../assets/images/host-raus.webp";
-import MapView from "../components/MapView/MapView";
+import MapView from "../components/mapView/MapView";
 import mapViewSampleImg from "./../assets/map-view-sample.png";
-import ProductGallery from "../components/ProductGallery/ProductGallery";
-import PersonProfile from "../components/PersonProfile/PersonProfile";
+import ProductGallery from "../components/productGallery/ProductGallery";
+import PersonProfile from "../components/personProfile/PersonProfile";
```
Comment (miklesch) [inline src/pages/ProductPage.jsx:8]: should be the old version, on main the components starting with capital letters

Comment (006080) [reply src/pages/ProductPage.jsx:8]: When I downloaded a ripo from the beginning, it complained of the wrong path, so I just solved it to work on it.

Comment (006080) [reply src/pages/ProductPage.jsx:8]: I think in this file we can see the changes before they were solved. And I solved them already. Don't you think so?

**Pull request closed without merging**